app/commands/git/sync_pull_requests_reputation.rb

```diff
@@ -0,0 +1,13 @@
+module Git
+  class SyncPullRequestsReputation
+    include Mandate
+
+    def call
+      ::Git::PullRequest.find_each do |pr|
+        User::ReputationToken::AwardForPullRequest.(pr.data[:action], pr.author, pr.data)
```

Comment:
We go over each (previously imported) pull request and try to award reputation for it. The data in `pr.data` is stored in a JSON field in the database, and we can pass it to `AwardForPullRequest` _as is_ due to us storing the data in the exact same format that the PR update webhook flow calls it.

Reply:
I think we should only go over ones that haven't already had reputation assigned. So I'd add a boolean to `Git::PullRequest` for `reputation_assigned` and then use that as a scope in the query here (e.g. `Git::PullRequest.reputation_not_assigned.find_each ...`)